scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -19,12 +19,19 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
-# Contains the tuples of all files that bypasses angular's security mechanisms.
+# Contains the tuples of all files that bypasses Angular's security mechanisms.
 # Please keep the list in alphabetical order.
 # NOTE TO DEVELOPERS: DO NOT ADD ANY NEW FILES TO THE TUPLES WITHOUT ASKING
 #  @seanlip FIRST. You will have to explain the usage in the PR and the PR may
 #  have a longer review time. This is because the reviewers will have to ensure
 #  that the introduction of bypassSecurityTrust was the only option.
 
-EXCLUDED_BYPASS_SECURITY_TRUST_FILES = ()
+EXCLUDED_BYPASS_SECURITY_TRUST_FILES = (
+    # SVG is treated as unsafe by default by Angular. In order to show SVGs, we
+    # have to manually sanitize the SVG and mark the value as safe. Marking a
+    # value as a safe is done by bypassing the inbuilt Angular's security
+    # mechanism. The svg-sanitizer file is going to be a permanent member in
+    # this list due to the aforementioned reason.
+    'core/templates/services/svg-sanitizer.service.spec.ts',
```

Comment:
Does the spec also need to be trusted, or just the production component?

Reply:
This is a part of #11306.
See https://github.com/oppia/oppia/pull/11850#issuecomment-786942336 for details.